app/src/main/java/edu/uwstout/p2pchat/ExpandableSettingsListViewAdapter.java

```diff
@@ -94,18 +94,19 @@ public ExpandableSettingsListViewAdapter(final Context context, Activity current
 
 
         //Customize info here.
-        final int PARENT_TEXT_SIZE = 38;
-        final int CHILD_TEXT_SIZE = 30;
+        final int PARENT_TEXT_SIZE = 30;
+        final int CHILD_TEXT_SIZE = 25;
 
         //Text Colors here.
-        final int textColorChild = Color.BLUE;
-        final int textColorParent = Color.GREEN;
+        final int textColorChild = Color.GRAY;
+        final int textColorParent = Color.BLACK;
```

Comment:
This code makes it look like you are declaring the entire layout programatically. You probably should have done it using XML layouts to remove these constants from the code.